### PR TITLE
changes to my css

### PR DESCRIPTION
**Pull request by khadijahMu**
Added media query and also added render link to my main branch

### CODE REVIEW

**Review by Xfarouq — CHANGES_REQUESTED** [review]
Always make sure to removes spaces in your code for best practice

**Comment by khadijahMu** [comment]
Sir what part of the figma design isn't the same as my website?

**Comment by khadijahMu** [comment]
i have checked the figmadesign and made the proper correction which included:
1.changing the hover color on my active cards
2.alignment of some paragraphs on mobile view
and also worked on my PR  commit comments and description

**Comment by Xfarouq** [comment]
Hi @khadijahMu ,

Your project is complete! There is nothing else to say other than... it's time to merge it!!!
Congratulations! 🎉

<div style="text-align:center;">
    Remote Collaboration
</div>

### Highlights
- ✅ **Professional Readme**
- ✅ **Good use of Gitflow**
- ✅ **UI looks like the Figma design**
- ✅ **Good use of HTML semantics**
- ✅ **Professional commit messages**
- ✅ **Professional PR title**
- ✅ **Professional PR description**

### Optional Suggestions

- Always make sure not to leave spaces in your code for best practices

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Review by Xfarouq — APPROVED** [review]
Hi @khadijahMu  ,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Good use of Html Semantics** 👍

### Required Changes ♻️

- **Your website isn't the same as the figma design**
- **Make sure your travel website is the same as the figma design**
- **Make your PR commit  and description to be professional**



### Optional Suggestions


- Remove whitespaces to make your code more professional


Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by khadijahMu** [comment]
Thank you sir!

File: styles.css [hunk 1]
```diff
@@ -804,30 +963,48 @@
...
+
+    .social {
+        justify-content: flex-start; 
+        gap: 32px; 
+    }
+
+    .home-link {
+        justify-content: flex-end;
+    }
+}
```
Comment (Devsalmasaeed) [inline styles.css:1131]: Hi @khadijahMu ,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️

- Avoid making the website to overflow

[- check](https://www.loom.com/share/775a9ea7ad1e4422b55f84431f4e69e4?sid=e8a8a129-afd1-4dd4-bc16-296b72feab2c).

- Remove the hover effect on all the cards

### 1. Remove the hover effect on each card

Step 1 screenshot

### 2. This is how your cards should be display

Step 2 screenshot

<br/>

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Review by Devsalmasaeed — CHANGES_REQUESTED** [review]

File: styles.css (hunk 1 above)
Comment (khadijahMu) [reply styles.css:1131]: i have updated the required changes. Thankyou!

**Review by Devsalmasaeed — APPROVED** [review]
Hi @khadijahMu ,

Your project is complete! There is nothing else to say other than... it's time to merge it!!!
Congratulations! 🎉

<div style="text-align:center;">
    Remote Collaboration
</div>

### Highlights
- ✅ **Professional Readme**
- ✅ **Good use of Gitflow**
- ✅ **UI looks like the Figma design**
- ✅ **Good use of HTML semantics**
- ✅ **Professional commit messages**
- ✅ **Professional PR title**
- ✅ **Professional PR description**

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Comment by khadijahMu** [comment]
Thank you so much for the great news!

**Pull request merged**